.github/workflows/safe-apps-check.yml

```diff
@@ -0,0 +1,44 @@
+name: Test Safe Apps
+on:
+  workflow_dispatch:
```

Comment:
I assume this event was for development purposes

Reply:
yep!